app/javascript/dashboard/routes/dashboard/helpcenter/components/HelpCenterLayout.vue

```diff
@@ -28,13 +28,14 @@ export default {
     UpgradePage,
     WootKeyShortcutModal,
   },
-  mixins: [portalMixin],
   setup() {
     const { uiSettings, updateUISettings } = useUISettings();
+    const { accountId } = usePortal();
```

Comment:
This shouldn't be in the `usePortal`, we can let it stay in the computed or use `useAccount` instead